src/components/Editor.tsx

```diff
@@ -78,13 +78,12 @@ export const Editor = ({
     devLogger.logData("DOCUMENT", document);
   }
 
-  const buildLocalStorageKey = useCallback(() => {
-    if (!templateMetadata) {
+  const buildVisualConfigLocalStorageKey = useCallback(() => {
+    if (!templateMetadata || templateMetadata.isThemeMode) {
```

Comment:
Don't we still need the visualConfiguration data from localStorage even if you're in theme mode?

Reply:
Don't think so, I just prevented those fetches from occurring when in theme mode

Reply:
None of the VConfig stuff should be affected by theme. We need it always.